### DISCUSSION TITLE
维度值竖向显示的时候能否中文也竖向显示

### DISCUSSION THREAD

**Discussion by qtj4js** [antvis/G2#6189] [Q&A]
![企业微信截图_17137709279479](https://github.com/antvis/G2/assets/47909978/4e5324db-8fb8-4331-b88a-08e5c81d6bdb)
希望能够这样显示
![企业微信截图_17137708784157](https://github.com/antvis/G2/assets/47909978/a3cf46e9-9d41-40d7-ba21-0f2cd28bc983)
是否可配置

**Comment by hustcc** [comment]
中间穿插 \n 即可自动换行。

**Reply by pearmini** [reply]
```js
chart.axis('x', { 
  labelFormatter: d => isChinese(d) ? d.split('').join('\n') : d , 
  labelTransform: 'rotate(0)'
})
```

**Reply by qtj4js** [reply]
虽然我想说要是中英文混合的时候,能把中文竖向显示,英文横向显示,但是我觉得能做到这样也算可以了?

**Reply by pearmini** [reply]
labelTransform 应该也是支持回调的：

```js
chart.axis('x', { 
  labelFormatter: d => isChinese(d) ? d.split('').join('\n') : d , 
  labelTransform: d => isChinese(d.label) ? 'rotate(0)': undefined
})
```

**Reply by qtj4js** [reply]
![企业微信截图_17137746685417](https://github.com/antvis/G2/assets/47909978/fee7b496-30a6-44a5-9a22-b051b4382f66)
我说的其实是这种情况,一个值里面有中文和英文.但是这个应该是没法实现了

**Reply by hustcc** [reply]
这个就看你怎么拆分字符串，加上 \n 了。
